packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -86,16 +87,27 @@ export class AtomicCategoryFacet
    * The number of values to request for this facet. Also determines the number of additional values to request each time this facet is expanded, and the number of values to display when this facet is collapsed.
    */
   @Prop() public numberOfValues = 5;
+  /**
+   * Whether this facet should contain a search box.
+   */
+  @Prop() public enableFacetSearch = false;
+  /**
+   * The sort criterion to apply to the returned facet values. Possible values are 'alphanumeric', and 'occurrences''.
```

Comment:
Would there be a benefit of mentioning the default in the tsdoc annotation? e.g `@default "occurences"`

Reply:
The docs defaults are filled in from the props default value.